tools/release.sh

```diff
@@ -20,7 +20,7 @@ signcmd=dist-sign
 
 echo "# Selecting GPG key ..."
 
-gpgkey=$(gpg --list-secret-keys | grep '^sec' | awk -F'( +|/)' '{print $3}')
+gpgkey=$(gpg --list-secret-keys | grep -E '^sec|^ssb' | awk -F'( +|/)' '{print $3}')
```

Comment:
I'm not 100% we want to be using the -E flag here. Personally I don't see a problem, but perhaps we want to solve this without using an extended regular expression